sdb/commands/stacks.py

```diff
@@ -128,28 +128,27 @@ def task_struct_get_state(task: drgn.Object) -> str:
     # of the current frame.
     #
     @staticmethod
-    def find_module_memory_segment(prog: drgn.Program,
-                                   mod_name: str) -> Tuple[int, int]:
+    def find_module_memory_segment(mod_name: str) -> Tuple[int, int]:
         """
         Looks for the segment in memory where `mod_name` is
-        loaded within `prog`.
+        loaded.
 
         Returns:
             (<base_offset>, <size>) if `mod_name` is found.
             (-1, 0) otherwise.
         """
         for mod in list_for_each_entry('struct module',
-                                       prog['modules'].address_of_(), 'list'):
+                                       sdb.prog['modules'].address_of_(),
+                                       'list'):
```

Comment:
It's ok if we don't change this in the current commit but it would be cleaner to explicitly pass `prog` as an argument to static methods instead of referencing global state within them. Occurrences of this in our codebase are not as severe in this scenario as we are probably not going to mutate `prog`, just read from it.

In this current case the reason why I made this a static method was that it seems like it can be potentially re-used by other commands so it may be eventually extracted in a module outside of the `sdb.Command` hierarchy (e.g. like `slub_helpers`).

Again, we don't need to do this in this commit (we can wait until such a consumer shows up),  I just wanted to bring this up for future reference.